collect/collect.go

```diff
@@ -47,6 +47,9 @@ var (
 	// Whether or not to use NTLM authentication
 	UseNtlm bool = false
 
+	// DefaultClient can be used to override the HTTP client that will be used to make requests.
+	DefaultClient *http.Client
```

Comment:
I'd rather collect never had to know anything about http clients, and can just Get and Post to its heart's content.